src/outputs/parthenon_hdf5.cpp

```diff
@@ -77,6 +81,32 @@ void HDF5WriteAttribute(const std::string &name, const std::vector<bool> &values
   HDF5WriteAttribute(name, values.size(), data.get(), location);
 }
 
+// template to get HDF5 environment variables
+template<typename T>
+static T get(const char* name, T DefaultVal, bool exists) {
+  exists = true;
+  char* value = std::getenv(name);
+
+  // Environment variable is not set
+  if (!value) {
+    exists = false;
+    return DefaultVal;
+  }
+
+  T res;
+  // Environment variable is set but no value is set, use the default
+  if( !value[0]) {
+    return DefaultVal;
+  } else {
+    // Environment variable is set and value is set
+    if (std::is_same<T, char*>::value)
+      res = (T)value;
+    else
+      std::istringstream(value) >> res;
+    return res;
+  }
+}
+
```

Comment:
This doesn't appear HDF5-specific to me. Perhaps we should move it into our utilities header or something?

Reply:
Correct, nothing generally specific for HDF5. This could be moved.

Reply:
I like putting this [here](https://github.com/lanl/parthenon/blob/develop/src/utils/utils.hpp). But it's also fine to leave it where it is.

Reply:
moved.